src/Symfony/Bundle/FrameworkBundle/Routing/Router.php

```diff
@@ -154,6 +155,8 @@ private function resolve($value)
 
             if (is_string($resolved) || is_numeric($resolved)) {
                 return (string) $resolved;
+            } elseif ($acceptArrayParameters && is_array($resolved)) {
```

Comment:
You did not import `$acceptArrayParameters` to the variable scope of the anonymous function. This expression will always evaluate to false and throw a notice.